### DISCUSSION TITLE
Add Domain/RegEx column in Query Log

### DISCUSSION THREAD

**Discussion by itguy327** [pi-hole/pi-hole#4661] [Bugs]
Not a bug, but wanted to know if it was possible to add the Domain/RegEx that was responsible for a block or allow as a column in the query log.

**Comment by jfb-pihole** [comment]
Feature requests are welcomed on our discourse site. 

You can also see which regex blocks a domain with the following command from the Pi terminal:

`pihole -q domain-name-here`

Example:

```
pihole -q analytics.ccs.mcafee.com
 Match found in regex blacklist
   ^analytics?[-.]
```

**Comment by yubiuser** [comment]
In the query log, you can click on "Blocked (regex blacklist)" and it will show you the regex that was responsible for blocking

**Reply by itguy327** [reply]
Thank you
